CHANGELOG.md

```diff
@@ -1,3 +1,237 @@
+
+#### Documentation
+
+- Add OpenEBS update guide ([#1163](https://github.com/kinvolk/lokomotive/pull/1163)).
+- Add Rook Ceph update guide ([#1165](https://github.com/kinvolk/lokomotive/pull/1165)),
+
+#### Miscellaneous
+
+- Pull control plane images from Quay to avoid hitting Docker Hub pulling limits ([#1226](https://github.com/kinvolk/lokomotive/pull/1226)).
+- Bootkube now waits for all control plane charts to converge before exiting, which should make the bootstrapping process more stable ([#1085](https://github.com/kinvolk/lokomotive/pull/1085)).
+
+### Updating from v0.5.0
+
+#### Configuration syntax changes
+
+##### AWS
+
+Removed the undocumented `cluster.os_name` parameter, since Lokomotive supports Flatcar Container Linux only.
+
+##### Bare-metal
+
+The `cluster.os_channel` parameter got simplified by removing the `flatcar-` prefix.
+
+###### Old
+
+```hcl
+os_channel = "flatcar-stable"
+```
+
+###### New
+
+```hcl
+os_channel = "stable"
+```
+
+##### Velero
+
+Velero requires an explicit `provider` field to select the provider.
+Example:
+
+```hcl
+component `velero` {
+  provider = "openebs"
+
+  openebs {
+    ...
+  }
+}
+```
+
+#### Cluster update steps
+
+> **NOTE**: Updating multiple Lokomotive versions at a time is not supported. If your cluster is running a version older than `v0.5.0`, upgrade to `v0.5.0` first and only then proceed with the upgrade to `v0.6.0`.
+
+Please perform the following manual steps in your cluster configuration directory.
+
+1. Download the release bundle
+
+  ```bash
+  curl -LO https://github.com/kinvolk/lokomotive/archive/v0.6.0.tar.gz
+  tar -xvzf v0.6.0.tar.gz
+  ```
+
+2. Install the Packet CCM
+
+  If you are running Lokomotive on Equinix Metal (formerly Packet), then install Packet CCM. Export your Packet cluster's project ID and API Key.
+
+  ```bash
+  export PACKET_API_KEY=""
+  export PACKET_PROJECT_ID=""
+
+  echo "apiKey: $PACKET_API_KEY
+projectID: $PACKET_PROJECT_ID" > /tmp/ccm-values.yaml
+
+  helm install packet-ccm --namespace kube-system --values=/tmp/ccm-values.yaml ./lokomotive-0.6.0/assets/charts/control-plane/packet-ccm/
+  ```
+
+3. Update all nodes
+
+  This script shipped with release tarball will update all the nodes to run the latest kubelet, latest etcd and add permanent MetalLB labels.
+
+  ```bash
+  ./lokomotive-0.6.0/scripts/update/0.5-0.6/update.sh
+  ```
+
+4. Export assets directory
+
+  ```sh
+  export ASSETS_DIR="assets"
+  ```
+
+5. Remove BGP sessions from Terraform state
+
+  If you are running Lokomotive on Equinix Metal (formerly Packet), then run the following commands:
+
+  ```bash
+  cd $ASSETS_DIR/terraform
+  export WORKER_POOL=""
+  export WORKER_COUNT=""
+
+  for ((i=0;i<$WORKER_COUNT;i++)); do
+    terraform state rm "module.worker-$WORKER_POOL.packet_bgp_session.bgp[$i]"
+  done
+  cd -
+  ```
+
+  Repeat above steps for each worker pool which has BGP enabled.
+
+6. Reload the new config for self-hosted kubelet
```

Comment:
```suggestion
6. Reload the new config for self-hosted kubelet:
```

Reply:
I removed all the colons https://github.com/kinvolk/lokomotive/pull/1292#discussion_r550126317

Reply:
If we don't use colons, should we put dots at the end of the sentences then?